src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js

```diff
@@ -74,6 +77,33 @@ const ContentExplorerActionButtons = ({
         }
     };
 
+    const getStatusElement = statusMessage => {
+        let statusElement = <span className="status-message">{statusMessage}</span>;
+
+        if (onViewSelectedClick) {
+            statusElement = (
+                <PlainButton
+                    className="status-message-include-subfolders"
+                    onClick={() => {
+                        const foldersPath = onViewSelectedClick();
+                        if (foldersPath && onFoldersPathUpdated) {
```

Comment:
i missed this, we can remove `onFoldersPathUpdated` from this check since it'll always be true